sdk/tests/client/node_api.rs

```diff
@@ -402,3 +402,19 @@ async fn test_mqtt() {
     }
     client.subscriber().disconnect().await.unwrap();
 }
+
+#[ignore]
+#[tokio::test]
+async fn test_call_plugin_route() {
+    let c = setup_client_with_node_health_ignored().await;
+
+    // we call the "custom" plugin "node info"
+    let plugin_res: NodeInfoWrapper = c
+        .call_plugin_route("/api/core/v2/", "GET", "info", vec![], None)
```

Comment:
```suggestion
        .call_plugin_route("api/core/v2/", "GET", "info", vec![], None)
```
does it need to be this?

Reply:
yea i tested it, its required